js_modules/dagster-ui/packages/ui-core/src/assets/auto-materialization/AutomaterializationEvaluationHistoryTable.tsx

```diff
@@ -142,38 +125,67 @@ export const AutomaterializationEvaluationHistoryTable = ({
   );
 };
 
-const StatusLabels = {
-  [InstigationTickStatus.SKIPPED]: 'None requested',
-  [InstigationTickStatus.STARTED]: 'Started',
-  [InstigationTickStatus.FAILURE]: 'Failed',
-  [InstigationTickStatus.SUCCESS]: 'Requested',
+export enum AutomaterializationTickStatusDisplay {
+  ALL = 'all',
+  FAILED = 'failed',
+  SUCCESS = 'success',
+}
+export const AutomaterializationTickStatusDisplayMappings = {
+  [AutomaterializationTickStatusDisplay.ALL]: [
+    InstigationTickStatus.SUCCESS,
+    InstigationTickStatus.FAILURE,
+    InstigationTickStatus.STARTED,
+    InstigationTickStatus.SKIPPED,
+  ],
+  [AutomaterializationTickStatusDisplay.FAILED]: [InstigationTickStatus.FAILURE],
+  [AutomaterializationTickStatusDisplay.SUCCESS]: [InstigationTickStatus.SUCCESS],
 };
 
-function StatusCheckbox({
+const StatusFilter = ({
   status,
-  statuses,
-  setStatuses,
+  onChange,
 }: {
-  status: InstigationTickStatus;
-  statuses: Set<InstigationTickStatus>;
-  setStatuses: (statuses: Set<InstigationTickStatus>) => void;
-}) {
+  status: AutomaterializationTickStatusDisplay;
+  onChange: (value: AutomaterializationTickStatusDisplay) => void;
+}) => {
+  const items = [
+    {key: AutomaterializationTickStatusDisplay.ALL, label: 'All ticks'},
+    {key: AutomaterializationTickStatusDisplay.SUCCESS, label: 'Requested'},
+    {key: AutomaterializationTickStatusDisplay.FAILED, label: 'Failed'},
+  ];
```

Comment:
Could make this static outside the component so that it's not re-created on every render.